sulong/myDebugTests/attr_t_debug_test.c

```diff
@@ -0,0 +1,36 @@
+#include <pthread.h>
```

Comment:
Copyright header missing. `mx` might be able to help you: try `mx checkcopyrights --primary -- --fix` in the sulong directory. (but be sure to save/commit all changes before ;))